packages/libs/client/jest.integration.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Does it make sense to extend from `packages/tools/heft-rig/jest.config.json` (and overwrite as needed like below), or not (yet)?

Reply:
I thought of that, but there are quite a few settings in the central config should not be used for the integration config. It might make sense to eventually add a shared integration config if more packages perform similar tests.